index.js

```diff
@@ -45,12 +48,17 @@ module.exports = function (config, windowParams) {
     var url = config.authorizationUrl + '?' + queryString.stringify(urlParams);
 
     return new Promise(function (resolve, reject) {
-      let authWindow = new BrowserWindow(windowParams || {'use-content-size': true});
+      if(authWindow){
+        authWindow.show();
```

Comment:
Don't we need to reload the url as well?

Reply:
Yeah, I had just test & saw we need call loadUrl to build some localstorage that we cleared from client app